motr/st/utils/helper.c

```diff
@@ -536,6 +546,18 @@ int m0_read(struct m0_container *container,
 	if (rc != 0)
 		goto get_error;
 
+	/* Setting pver here to read_pver received as parameter to this func.
+	 * Caller of this function is expected to pass pver of object tobe
+	 * read, if he knows pver of object.
+	 * */
+	if (read_pver != NULL &&  m0_fid_is_set(read_pver)) {
+		obj.ob_attr.oa_pver.f_container = read_pver->f_container;
+		obj.ob_attr.oa_pver.f_key = read_pver->f_key;
+		M0_LOG(M0_DEBUG, "m0_read: obj->ob_attr.oa_pver is set to:"FID_F
+                       " To predent that, it is received from s3",
```

Comment:
I propose don't mention "s3".
Cortx-s3server is a user of this cortx-motr.
But from Motr itself, we don't know or we don't care who is the user of Motr APIs.
Any Motr application can do this.